src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -39,335 +29,257 @@
-        else:  # Retrieve delegation for existing job
-            jj = {"job": [{"id": jobID}]}  # job in ARC REST json format
+
+                    # Submit the proxy
+                    response = self.session.put(query, data=response.text, headers=self.headers, timeout=self.arcRESTTimeout)
+                    if not response.ok:
+                        self.log.warn(
+                            "Issue while interacting with the delegation",
+                            "%s - %s" % (response.status_code, response.reason),
+                        )
+                        delegationID = ""
+
+            return S_OK(delegationID)
+
+        # Retrieve delegation for existing job
+        else:
+            # Prepare the command
             command = "jobs"
             params = {"action": "delegations"}
             query = self.base_url + command
-            r = self.s.post(query, data=json.dumps(jj), headers=self.headers, timeout=self.arcRESTTimeout)
-            dID = ""
-            if r.ok:  # Check if the job has a delegation
-                p = json.loads(r.text.replace("\n", ""))
+
+            # Submit the POST request to get the delegation
+            jobsJson = {"job": [{"id": jobID}]}
+            response = self.session.post(
+                query, data=json.dumps(jobsJson), headers=self.headers, timeout=self.arcRESTTimeout
+            )
+            delegationID = ""
+            if response.ok:  # Check if the job has a delegation
+                p = response.json()
                 if "delegation_id" in p["job"]:
-                    dID = p["job"]["delegation_id"][0]
-            return dID
+                    delegationID = p["job"]["delegation_id"][0]
+            return S_OK(delegationID)
 
     #############################################################################
+
     def submitJob(self, executableFile, proxy, numberOfJobs=1):
         """Method to submit job
         Assume that the ARC queues are always of the format nordugrid-<batchSystem>-<queue>
         And none of our supported batch systems have a "-" in their name
         """
-        if not self.initialised:  # Something went wrong in the initialisation. Redo it.
+        if not self.session:
             return S_ERROR("REST interface not initialised. Cannot submit jobs.")
+        self.log.verbose("Executable file path: %s" % executableFile)
 
+        # Get the name of the queue: nordugrid-<batchsystem>-<queue>
         self.arcQueue = self.queue.split("-", 2)[2]
 
+        # Get a proxy
         result = self._prepareProxy()
         if not result["OK"]:
-            self.log.error("AREXComputingElement: failed to set up proxy", result["Message"])
+            self.log.error("Failed to set up proxy", result["Message"])
             return result
-        self.s.cert = Locations.getProxyLocation()
-
-        self.log.verbose("Executable file path: %s" % executableFile)
-        if not os.access(executableFile, 5):
```

Comment:
Could you motivate the removal of these lines please.

Reply:
For my understanding, what do you mean by "it is handled by the ARC instance automatically"?

Setting the permissions was done also in the ARCComputingElement, as it was needed at that time. If this is now done automatically by DIRAC, fine...

Reply:
> Yes - ARCComputingElement will come once we have AREXComputingElement fixed to our comfort.

Well but this [issue](https://github.com/DIRACGrid/DIRAC/issues/6260) is related to `ARCComputingElement`.
By removing the method in `ARCComputingElement`, we fix the issue.

> Setting the permissions was done also in the ARCComputingElement, as it was needed at that time. If this is now done automatically by DIRAC, fine...

Yes, this should be done by the ARC instance itself.
Let me try that a last time with the Gridftp and AREX services. If everything is okay without the permission, then I remove the code from both `AREXComputingElement` and `ARCComputingElement`.